zone/inventory.cpp

```diff
@@ -1673,7 +1673,7 @@ bool Client::SwapItem(MoveItem_Struct* move_in) {
 			if (dstbag)
 				dstbagid = dstbag->GetItem()->ID;
 		}
-		if (srcitemid==17899 || srcbagid==17899 || dstitemid==17899 || dstbagid==17899){
+		if ((srcbagid && srcbag->GetItem()->BagType == EQ::item::BagTypeTradersSatchel) || (dstbagid && dstbag->GetItem()->BagType == EQ::item::BagTypeTradersSatchel) || (srcitemid && src_inst->GetItem()->BagType == EQ::item::BagTypeTradersSatchel) || (dstitemid && dst_inst->GetItem()->BagType == EQ::item::BagTypeTradersSatchel)){
```

Comment:
Breaking each OR condition into a separate line would make this more readable

```cpp
if ((srcbagid && srcbag->GetItem()->BagType == EQ::item::BagTypeTradersSatchel) ||
    (dstbagid && dstbag->GetItem()->BagType == EQ::item::BagTypeTradersSatchel) ||
    (srcitemid && src_inst->GetItem()->BagType == EQ::item::BagTypeTradersSatchel) ||
    (dstitemid && dst_inst->GetItem()->BagType == EQ::item::BagTypeTradersSatchel)) {
```

Reply:
Works for me

Reply:
@N0ctrnl were you going to push the change for this prior to merge?

Reply:
My bad. Done.

Reply:
@N0ctrnl no problem at all man! Thanks for pushing this <3